db/seeds.rb

```diff
@@ -119,7 +119,7 @@
 
 interest_ids = []
 Interest.all.each do |interest|
-  interest_ids.push(interest.id)
+  interest_ids.push(interest.name)
```

Comment:
This line is putting the string value of interest.name into the array that was meant for interest IDs.

On line 128, the interest_ids array is sampled from to generate interests for users by ID. This is important and should not be changed or else the seed file will fail.